index.html

```diff
@@ -0,0 +1,133 @@
+<!DOCTYPE html>
+<html lang="en-us">
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="stylesheet" href="./assets/css/style.css" />
+    <title>Horiseon: Leading Marketing Agency</title>
+  </head>
+
+  <body>
+    <!-- header section with navbar containing links -->
+    <header class="navbar">
+      <h1>Hori<span class="seo">seo</span>n</h1>
+      <!-- navbar containing links -->
+      <nav>
+        <ul>
+          <li>
+            <a href="#search-engine-optimization">Search Engine Optimization</a>
+          </li>
+          <li>
+            <a href="#online-reputation-management"
+              >Online Reputation Management</a
+            >
+          </li>
+          <li>
+            <a href="#social-media-marketing">Social Media Marketing</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <!-- insert of team photo from FY2021 -->
+
+    <img
+      class="hero"
+      src="./assets/images/digital-marketing-meeting.jpg"
+      alt="business people in a meeting discussing reports"
+    />
+
+    <!-- main content section about key services offered to clients -->
+    <main class="content">
+      <section class="content-box" id="search-engine-optimization">
+        <img
+          src="./assets/images/search-engine-optimization.jpg"
+          class="float-left"
+          alt="On top of a wooden table, there is a notepad, pencils a laptop with a magnifying glass and some coffee"
+        />
+
```

Comment:
Some of your `alt` tags are quite long but they don't have to be. You could shorten them but there is nothing wrong with this.

Reply:
Did think it was quite long, thanks for the advice!